src/lib.rs

```diff
@@ -501,6 +530,15 @@ impl Deref for System {
     }
 }
 
+/// What can be received during one actor event loop.
+enum Received<M> {
```

Comment:
nit: Did you struggle with the naming? There is something a bit confusing about it. There is `Recipient` already, but `Received` is not received by `Recipient` but by `Actor`. Not that I can think of a better name, but `ActorMessage`, `ActorSelectLoopMessage`, something along that line?

Reply:
> Did you struggle with the naming?

I did. :) I was able to move `Received<M>` into `run_actor_select_loop()` - it is its implementation detail. Do you think that is sufficient to resolve the confusing naming, or shall we still rename it?

Reply:
That's sufficient. Thank you @strohel :)